recipes/iso_varying_string/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "iso_varying_string" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://gitlab.com/everythingfunctional/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 7d003ec444ca68f74e62588449ebe29e40721e61b14b830046d11b176d31f912
+
+build:
+  number: 0
+  script:
+    - set -ex  # [unix]
+    - export FFLAGS="${FFLAGS} -fcoarray=single"  # [unix]
+    - set FFLAGS=-fcoarray=single  # [win]
+    - cmake ${CMAKE_ARGS} -DBUILD_SHARED_LIBS=ON -B_build -GNinja  # [unix]
+    - cmake -DCMAKE_INSTALL_PREFIX=%PREFIX% -DBUILD_SHARED_LIBS=ON -B_build -GNinja  # [win]
```

On windows, this should go into LIBRARY_PREFIX
```suggestion
    - cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DBUILD_SHARED_LIBS=ON -B_build -GNinja  # [win]
```